PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA.h

```diff
@@ -95,16 +97,23 @@ class AliVTrack;
 class AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA : public AliAnalysisTaskEmcalJet {
 public:
 
-    AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA()                                               ;
-    AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA(const char *name)                               ;
-    virtual ~AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA()                                      ;
+    AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA()                                 ;
+    AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA(const char *name)                 ;
+    virtual ~AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA()                        ;
 
     void                        UserCreateOutputObjects()                         ;
     void                        Terminate(Option_t *option)                       ;
     void                        ExtractMainPatch()                                ;
     
     //setters
     void                        SetUseSumw2(Bool_t b) { fUseSumw2          = b   ; }
+    
+    static AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA * AddTaskEmcalJetSpectra8TeVTriggerQA(
```

Comment:
I don't see the implementation for this - can you point me to it?

Reply:
The tests fail with an undefined symbol, `_ZN43AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA35AddTaskEmcalJetSpectra8TeVTriggerQAEPKcS1_S1_S1_S1_ `, which is related to your AddTask (I can't unmangle it at the moment, but you can with `c++filt`). You've only provided the declaration, but you need to define this function in your `.cxx`. That should be sufficient to fix your error. I also recommend that you convert your .C macro into a wrapper around that function, like the example shown [here](https://github.com/alisw/AliPhysics/blob/master/PWGJE/EMCALJetTasks/macros/AddTaskEmcalJetHCorrelations.C), but it is up to you